plugins/modules/azure_rm_subnet.py

```diff
@@ -343,16 +343,6 @@
     serviceName=dict(
         type='str',
         required=True,
-        choices=['Microsoft.Web/serverFarms', 'Microsoft.ContainerInstance/containerGroups', 'Microsoft.Netapp/volumes',
```

Comment:
Why delete the choices？  Please keep！

Reply:
@Fred-sun I am sorry. That is actually part of a different pull request I am planning. Let me figure out on how to have multipole pulls at the same time. Can you mean comment of the remaining open topics? I will revert this part later. Sorry again, I am no so strong with pull requests and its management.

Reply:
@Fred-sun Hi, I moved the change into a different pull https://github.com/ansible-collections/azure/pull/1414  .. We can go ahead on this one.